erpnext_retail/public/js/point_of_sale.js

```diff
@@ -1,28 +1,222 @@
+/* global Payment, __, frappe */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['frappe' is already defined as a built-in global variable.](https://app.codacy.com/gh/techlift-tech/erpnext_retail/pullRequest?prid=4838702)